lakeparse/head.go

```diff
@@ -12,13 +12,16 @@ type Commitish struct {
 }
 
 func ParseCommitish(commitish string) (*Commitish, error) {
+	if commitish == "" {
+		return nil, errors.New("empty pool and branch")
```

Comment:
Should this "branch" be "ref" since this could be a branch or commit sha? I do think "branch" is clearer though...

Reply:
Excellent question! I think we do want to change our terminology here, though "ref" doesn't feel quite right because in Git, a commit ID isn't a ref. What I'd like to do is commit this as is and improve it later.